Shared/Protocols/Statusable.swift

```diff
@@ -14,6 +14,7 @@ import Combine
 protocol Statusable: AnyObject {
     // Handles and returns the status in a future once it has been handled. Maybe async due to animations
     // This allows for API calls to be combined with animation blocks
+#warning("Convert to async")
```

Comment:
I added a bunch of warnings to make it easier to find all the remaining futures that should be converted.